src/Symfony/Component/Form/FormInterface.php

```diff
@@ -103,6 +103,24 @@ public function all();
      */
     public function getErrors($deep = false, $flatten = true);
 
+    /**
+     * Returns the list of the current form and all child forms.
+     *
+     * @return FormErrorIterator An iterator over the {@link FormError}
+     *                           instances that where added to the form
+     *                           and the child forms
+     */
+    public function getAllErrors();
+
+    /**
+     * Returns the list of this form only - without child errors.
+     *
+     * @return FormErrorIterator An iterator over the {@link FormError}
+     *                           instances that where added to the top
+     *                           level of this form
+     */
+    public function getOnlyGlobalErrors();
```

Comment:
The name sounds confusing for a form which is not the root.

Reply:
I was trying to stick with the naming in the current documentation:
```
// a FormErrorIterator instance, but only errors attached to this
// form level (e.g. global errors)
```
But `getRootErrors()` works for me too if this is a naming we can agree on? The suggested `getOwnErrors()` didn't appeal to me.